From 365fe8ce794aa561c58d69902945ba663f7bfb7a Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Tue, 07 Aug 2018 06:43:14 +0300
Subject: [PATCH] board [busybox]: add applets

Add watch and vi for convenience.
It adds about 20KB to FIT
---

diff --git a/board/habanalabs/dali/busybox.fragment b/board/habanalabs/dali/busybox.fragment
index 1721fd4..8edd83c 100644
--- a/board/habanalabs/dali/busybox.fragment
+++ b/board/habanalabs/dali/busybox.fragment
@@ -6,3 +6,20 @@
 CONFIG_DEFAULT_DEPMOD_FILE="modules.dep"
 CONFIG_DIFF=y
 CONFIG_CKSUM=y
+CONFIG_WATCH=y
+CONFIG_VI=y
+CONFIG_FEATURE_VI_MAX_LEN=4096
+CONFIG_FEATURE_VI_COLON=y
+CONFIG_FEATURE_VI_YANKMARK=y
+CONFIG_FEATURE_VI_SEARCH=y
+CONFIG_FEATURE_VI_USE_SIGNALS=y
+CONFIG_FEATURE_VI_DOT_CMD=y
+CONFIG_FEATURE_VI_READONLY=y
+CONFIG_FEATURE_VI_SETOPTS=y
+CONFIG_FEATURE_VI_SET=y
+CONFIG_FEATURE_VI_WIN_RESIZE=y
+CONFIG_FEATURE_VI_ASK_TERMINAL=y
+CONFIG_FEATURE_VI_UNDO=y
+CONFIG_FEATURE_VI_UNDO_QUEUE=y
+CONFIG_FEATURE_VI_UNDO_QUEUE_MAX=256
+CONFIG_FEATURE_ALLOW_EXEC=y
